utils/crypto.ts

```diff
@@ -164,3 +164,22 @@ export const electrumVaultMnemonicToSeed = (mnemonic: string, password = '') =>
     keylen: 64,
     digest: 'sha512',
   });
+
+export const getMasterPublicKeyPrefix = (header: MasterPublicKey) => {
```

Comment:
IMHO `networkName` should be passed in the arguments to minimalize code complexity and to keep functions pure as much as possible

Reply:
is more like a header in a complex app or if u wanna is key as a single parameter